leak-detect_test.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2013 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
```

Comment:
if this is a golang package why can't we directly reference it and vendorize it?